language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/tests/helpers/DummyLedgerClient.scala

```diff
@@ -109,12 +109,28 @@ class DummyLedgerClient(
         offset: LedgerOffset,
         parties: util.Set[String]): Flowable[CompletionStreamResponse] =
       commandCompletions
+    // TODO Untested endpoint
+    override def completionStream(
+        applicationId: String,
+        offset: LedgerOffset,
+        parties: util.Set[String],
+        accessToken: String): Flowable[CompletionStreamResponse] =
+      ???
     override def completionStream(
         applicationId: String,
         parties: util.Set[String]): Flowable[CompletionStreamResponse] =
       commandCompletions
+    // TODO Untested endpoint
```

Comment:
Same here. Perhaps extract it out into a function named `endpointUnusedInTests` or something.

Reply:
I'm trying to move the clients in parallel and this is a single file that everybody touches, I'll leave it alone for the time being but I will definitely apply this suggestion when I'm putting all the clients together in the final PR for this ticket. Thanks! :+1: